src/Components/DrillAnimationPage.js

```diff
@@ -1,18 +1,16 @@
 import React, { Component } from 'react';
-import { StyleSheet, View, Text } from 'react-native';
+import { StyleSheet, View, Text, Dimensions } from 'react-native';
 import Animation from './Animation';
 import { WebView } from 'react-native-webview';
 
-class DrillPage extends Component {
+class DrillAnimationPage extends Component {
   render() {
-    const drill = this.props.route.params.drill;
-
     return (
       <View style={styles.drillAnimationPage}>
-        {drill.animation ? (
-          <Animation animation={drill.animation} />
-        ) : drill.video ? (
-          <WebView source={{ uri: drill.video }} style={styles.videoPresentation} />
+        {this.props.animation ? (
```

Comment:
Maybe extract the props at the beginning of the render method ?

And switch to a functional component... Or even remove this componentt ? It does nothing but a `if/else`. It could be done directly on DrillPage

Reply:
We are going to change it in the branch vimeovideo. Could we keep it ?

Reply:
Yep 👍